ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -84,16 +87,48 @@ static boolean bug1602844Affected(Map<String, String> attributes) {
    * Redact fields that may contain unintended sensitive information, replacing with null or
    * other appropriate signifiers.
    */
-  public static void redact(Map<String, String> attributes, ObjectNode json) {
+  public static void redact(Map<String, String> attributes, ObjectNode json)
+      throws MessageShouldBeDroppedException, AffectedByBugException {
     if (bug1602844Affected(attributes)) {
       json.path("events").elements().forEachRemaining(event -> {
         JsonNode eventMapValues = event.path(5);
         if (eventMapValues.has("fxauid")) {
           ((ObjectNode) eventMapValues).replace("fxauid", NullNode.getInstance());
-          countRedactedBug1602844.inc();
+        }
+
+        try {
+          handleBug("1602844", ScrubAction.REDACT);
+        } catch (Exception e) {
```

Comment:
I can't think of a situation where handleBug would throw an exception other than the two checked ones, so this is probably fine. But as a rule, it's dangerous to catch `Exception` generically and drop it.

This line should catch only MessageShouldBeDroppedException and AffectedByBugException